app.js

```diff
@@ -173,7 +173,12 @@ client.on("message", (msg) => {
   if (msg.author.bot) return;
 
   if (msg.content === "get help") {
-    msg.channel.send({ embeds: [exampleEmbed] });
+    msg.channel.send({ embeds: [exampleEmbed] })
+    .then((embed) => {embed.react("⬆"),
```

Comment:
We don't need these reactions for the help message right?

Reply:
okk, I will remove it.